src/main/java/com/murgaloids/server/entities/Student.java

```diff
@@ -0,0 +1,69 @@
+package com.murgaloids.server;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Table(name = "students")
+public class Student {
+  @Id
+  @GeneratedValue(strategy=GenerationType.AUTO)
+  private Integer id;
+
+  @Column(nullable = false)
+  private String email;
+
+  @Column(nullable = false)
+  private String firstName;
+
+  @Column(nullable = false)
+  private String lastName;
+
+  protected Student() {/** Used by JPA spec **/}
+
+  public Student(
+    String email,
```

Comment:
Why do we need getters and setters if the member variables are public?

Reply:
this is a constructor, it's not a public member variable.